src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/ConfigureTaskTest.java

```diff
@@ -114,7 +128,7 @@ private String createTask(CreateTaskMessage createTaskMessage) {
     }
 
```

Comment:
Can we include an FT to capture the first AC; I.e. a task can be configured but unassigned if it does not have any tribunal-caseworker case role is assigned to it?

Reply:
done. Please, double-check.